playground/AspireEventHub/EventHubs.AppHost/aspire-manifest.json

```diff
@@ -22,14 +22,18 @@
         "principalId": ""
       }
     },
+    "eventhub": {
```

Comment:
FYI - These are caused by other changes that didn't update the playground manifests.